### ISSUE TITLE
Is it possible to add multiple(alias) csv keys while unmarshaling

### ISSUE THREAD

**Issue by sandeepmeenuga** [gocarina/gocsv#116]
Is it possible to add multiple(alias) csv keys while unmarshaling.
like 	
eg: type Person struct {
		Name            string `csv:"PersonName,FirstName"`
		ID       string `csv:"UniqueID,ID"`
     }

so, csv column names could be either PersonName or FirstName can convert it as Name.

**Comment by ashishmohite** [comment]
Yes, It's possible as per the test https://github.com/gocarina/gocsv/blob/master/sample_structs_test.go#L62
